jsk_fetch_robot/jsk_fetch.rosinstall.melodic

```diff
@@ -59,12 +59,18 @@
 - git:
     local-name: jsk-ros-pkg/jsk_demos
     uri: https://github.com/jsk-ros-pkg/jsk_demos.git
-    version: master
+    version: develop/fetch
```

Comment:
Why did you also change the branch of jsk_demo?

Reply:
Sorry, I made a mistake...